using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -155,160 +168,182 @@ func application(_ application: UIApplication, didRegisterForRemoteNotifications
 {% endhint %}
 
 ### Swift
 
 **Syntax**
 
 ```swift
-func trackNotificationReceive(_ trackInfo: [String: String])
+func trackNotificationClick(_ trackInfo: [String: String])
 ```
 
 **Example**
 
 ```swift
-ACPCampaignClassic.trackNotificationReceive(trackInfo[String:String])
+func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+  guard let userInfo = response.notification.request.content.userInfo as? [String: String] else {
+    return;
+  }
+  ACPCampaignClassic.trackNotificationClick(userInfo);
+  completionHandler();
+}
 ```
 
 ### Objective-C
 
 **Syntax**
 
 ```objectivec
-+ (void) trackNotificationReceive: (nonnull NSDictionary<NSString*, NSString*>*) trackInfo;
++ (void) trackNotificationClick: (nonnull NSDictionary<NSString*, NSString*>*) trackInfo;
 ```
 
 **Example**
 
 ```objectivec
-- (void)application:(UIApplication *)application didReceiveRemoteNotification:(NSDictionary *)launchOptions fetchCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler 
+-(void)userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void(^)(void))completionHandler
 {
-    if ( launchOptions) NSLog(@"launchOptions: %@", [launchOptions description]);
-    // Tracking silent push notification receive
-    if ( [launchOptions[@"aps"][@"content-available"] intValue] == 1 ) {
-        NSLog(@"Silent Push Notification");
-        [ACPCampaignClassic trackNotificationReceive:launchOptions];
-        completionHandler(UIBackgroundFetchResultNoData);
-    }
+  NSLog(@"User Info : %@",response.notification.request.content.userInfo);
+  // Track action selected by the user for a given notification
+  [ACPCampaignClassic trackNotificationClick:response.notification.request.content.userInfo];
+  completionHandler();
 }
 ```
 
 {% endtab %}
 {% endtabs %}
 
-## trackNotificationClick
+## trackNotificationReceive
 
-The `trackNotificationClick` API sends the clicked push notification's tracking information to the configured Adobe Campaign Classic server. This API can be used to send tracking information when the notification is clicked, which may result in the application being opened. 
+The `trackNotificationReceive` API sends the received push notification's tracking information to the configured Adobe Campaign Classic server.
 
 {% tabs %}
 {% tab title="Android" %}
+
 ### Java
 
-{% hint style="info" %}
-If `trackInfo` is null, or does not contain the necessary tracking identifiers, `messageId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
-{% endhint %}
+If `trackInfo` is null or does not contain the necessary tracking identifiers, `messageId` (`_mId`) and `deliveryId` (`_dId`), a track request is **not** sent.
```

Comment:
Is there a reason we aren't wrapping this in a hint block?

Reply:
no, but it should be. i'll get it in a hint block.